python/src/Contour_doc.i.in

```diff
@@ -2,10 +2,13 @@
 "Contour.
 
 Available constructors:
-    Contour(*dimX, dimY, data, legend=' '*)
+
+    Contour(*dimX, dimY, data, isFilled=True, colorMap='viridis'*)
 
     Contour(*sampleX, sampleY, data, levels, labels, drawLabels=True, legend=' '*)
```

Comment:
maybe we should not mention default values here, it can give the feeling the kwargs are allowed

Reply:
As mentioned above, I have removed all optional arguments, so this problem is solved ;)